src/external/song.ts

```diff
@@ -17,100 +17,186 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import abortController from 'abort-controller';
+import { isArray } from 'lodash';
```

Comment:
no need to change - just letting you know - Array.isArray is a thing - no need for lodash